src/main/python/covata/delta/api/requestsclient.py

```diff
@@ -40,41 +43,117 @@ def register_identity(self, external_id=None, metadata=None):
         response = requests.post(
             url=self.DELTA_URL + self.RESOURCE_IDENTITIES,
             json=dict((k, v) for k, v in body.items() if v is not None))
-
-        identity_id = response.json()['identityId']
+        response.raise_for_status()
+        identity_id = response.json()['identityId']     # type: str
 
         self.keystore.store_keys(
             identity_id=identity_id,
             private_signing_key=private_signing_key,
             private_encryption_key=private_encryption_key)
         return identity_id
 
-    def get_identity(self, requestor_id, identity_id):
-        return requests.get(
+    def get_identity(self,
+                     requestor_id,  # type: str
```

Comment:
The method comments are inconsistent due to the change in structure to support type hinting in the IDE. I think this would read better following the contention of having the method comment and types under the method declaration. Not everyone will be necessarily using an IDE to read the code.